src/assets/less/pages/globalping/users.less

```diff
@@ -0,0 +1,212 @@
+
+		span {
+			font-family: Lexend, sans-serif;
+			font-size: 14px;
+			line-height: 20px;
+			color: #17233a;
+		}
+
+		&_header {
+			display: flex;
+			align-items: center;
+			column-gap: 4px;
+			margin-bottom: 8px;
+
+			&_location, &_cnt {
+				font-weight: 600;
+			}
+
+			&_cnt {
+				display: flex;
+				align-items: center;
+				justify-content: center;
+				height: 20px;
+				padding: 0 8px;
+				border: 1px solid #e7e7ee;
+				border-radius: 12px;
+				background: #e8fff9;
+			}
+		}
+
+		&_row {
+			display: flex;
+			align-items: center;
+			column-gap: 4px;
+
+			&_network-cnt {
+				display: inline-flex;
+				align-items: center;
+				justify-content: center;
+				font-size: 14px;
+				font-weight: 600;
+				line-height: 20px;
+				padding: 0 8px;
+				border: 1px solid #e7e7ee;
+				border-radius: 12px;
+				background-color: #fff;
+			}
+		}
+	}
+
+	.page-content {
+		display: flex;
+		flex-direction: column;
+		height: 100%;
+		column-gap: 24px;
+		width: 100%;
+		padding: 0 24px 40px;
+
+		@media (min-width: @screen-sm-min) {
+			padding: 0 15px 80px;
+			width: 1254px;
+			flex-direction: row;
+			justify-content: space-between;
+		}
+
+		&_side-menu {
+			display: flex;
+			flex-direction: column;
+			width: 100%;
+			margin-top: 24px;
+
+			@media (min-width: @screen-xs-max) {
+				width: 224px;
+			}
+
+			.menu-section {
+				display: flex;
+				flex-direction: column;
+
+				.section-list {
+					> div {
+						font-size: 20px;
+						font-weight: 600;
+						line-height: 24px;
+						color: #17233b;
+
+						&:not(:nth-last-child(1)) {
+							margin-bottom: 16px;
+						}
+					}
+				}
+			}
+		}
+
+		&_main {
+			width: 100%;
+
+			@media (min-width: @screen-sm-min) {
+				max-width: 912px;
+			}
+
+			&_section {
+				margin-top: 24px;
+				display: flex;
+				flex-direction: column;
+				gap: 4px;
+
+				&_title {
+					font-size: 20px;
+					font-weight: 600;
+					line-height: 24px;
+					margin-bottom: 12px;
+				}
+
+				&_probe {
+					border: 1px solid #e7e7ee;
+					border-radius: 12px;
+					display: grid;
+					grid-template-columns: 1fr;
+					gap: 16px;
+					padding: 8px 12px;
+
+					@media (min-width: @screen-sm-min) {
+						grid-template-columns: 1fr minmax(auto, 260px);
+					}
+
+					&_details {
+						display: flex;
+						align-items: center;
+						gap: 12px;
+
+						&_icon {
+							border: 1px solid #e0e0e0;
+							border-radius: 100%;
+							width: 32px;
+							height: 32px;
+							display: flex;
+							justify-content: center;
+							align-items: center;
+						}
+
+						&_info {
+							&_city {
+								display: flex;
+								align-items: center;
+								gap: 8px;
+								font-size: 14px;
+								font-weight: 600;
+							}
+
+							&_location {
+								font-size: 14px;
+							}
+						}
+					}
+
+					&_tag {
+						display: grid;
+						gap: 4px;
+
+						> span {
+							border-radius: 6px;
+							border: 1px solid #e0e0e0;
+							width: fit-content;
```

Comment:
If really needed you can add a comment like this to allow it: https://github.com/jsdelivr/www.jsdelivr.com/blob/21a30c9fa55566142b3d6d8bbadc049cf6097b6c/src/assets/less/pages/esm.less#L476

Reply:
That was added to prevent flex column's width calculations, with the new model it's not needed any more.